runtimes/polimec/src/lib.rs

```diff
@@ -918,7 +918,11 @@ where
 				Runtime,
 				pallet_asset_tx_payment::ChargeAssetTxPayment<Runtime>,
 			>::from(pallet_asset_tx_payment::ChargeAssetTxPayment::<Runtime>::from(tip, None)),
+			#[cfg(feature = "metadata-hash")]
 			frame_metadata_hash_extension::CheckMetadataHash::<Runtime>::new(true),
+			#[cfg(not(feature = "metadata-hash"))]
```

Comment:
How exactly does this work with metadata-hash? Shouldn't we always build the runtime with the new CheckmetadataHash? Was this only for testing purposes?

Reply:
![CleanShot 2024-12-17 at 16.13.00@2x.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/Rk8O2JJI4xgR8dtiq7Dh/5222bc15-9e62-4140-a112-d9524b107de3.png)